plugins/inappauthplugin/plugin.cpp

```diff
@@ -55,91 +55,90 @@ Daemon::Plugins::InAppPlugin::beginUserInputInteraction(
-                                      Q_ARG(QString, interactionServiceAddress),
-                                      Q_ARG(Sailfish::Secrets::Result, Sailfish::Secrets::Result(Sailfish::Secrets::Result::Succeeded)),
-                                      Q_ARG(QByteArray, QByteArray()));
-            return Result(Result::Pending);
-        } else if (interactionParameters.promptText().newInstruction().endsWith(
-                       QLatin1String("Enter the new lock code for the plugin."))) {
-            QMetaObject::invokeMethod(this, "userInputInteractionCompleted", Qt::QueuedConnection,
-                                      Q_ARG(uint, callerPid),
-                                      Q_ARG(qint64, requestId),
-                                      Q_ARG(Sailfish::Secrets::InteractionParameters, interactionParameters),
-                                      Q_ARG(QString, interactionServiceAddress),
-                                      Q_ARG(Sailfish::Secrets::Result, Sailfish::Secrets::Result(Sailfish::Secrets::Result::Succeeded)),
-                                      Q_ARG(QByteArray, QByteArray("pluginlock")));
-            return Result(Result::Pending);
-        } else if (interactionParameters.promptText().instruction().endsWith(
-                       QLatin1String("Enter the lock code to unlock the plugin."))) {
-            QMetaObject::invokeMethod(this, "userInputInteractionCompleted", Qt::QueuedConnection,
-                                      Q_ARG(uint, callerPid),
-                                      Q_ARG(qint64, requestId),
-                                      Q_ARG(Sailfish::Secrets::InteractionParameters, interactionParameters),
-                                      Q_ARG(QString, interactionServiceAddress),
-                                      Q_ARG(Sailfish::Secrets::Result, Sailfish::Secrets::Result(Sailfish::Secrets::Result::Succeeded)),
-                                      Q_ARG(QByteArray, QByteArray("pluginlock")));
-            return Result(Result::Pending);
-        } else if (interactionParameters.operation() != InteractionParameters::RequestUserData) {
-            QMetaObject::invokeMethod(this, "userInputInteractionCompleted", Qt::QueuedConnection,
-                                      Q_ARG(uint, callerPid),
-                                      Q_ARG(qint64, requestId),
-                                      Q_ARG(Sailfish::Secrets::InteractionParameters, interactionParameters),
-                                      Q_ARG(QString, interactionServiceAddress),
-                                      Q_ARG(Sailfish::Secrets::Result, Sailfish::Secrets::Result(Sailfish::Secrets::Result::Succeeded)),
-                                      Q_ARG(QByteArray, QByteArray("sailfish"))); // passphrase for importKey unit tests
-            return Result(Result::Pending);
-        }
+    if (interactionServiceAddress.isEmpty()
```

Comment:
was this change intentional?  Without looking deeper, it looks like the original intention was that the test plugin behaviour would only occur if the interaction service address is empty.

Reply:
Oh, I see, this is a reversion of the breaking change; never mind.